PORTAL/jobs/queue.py

```diff
@@ -46,7 +48,7 @@ class JobAlreadyQueuedError(QueuedJobError):
 class JobQueueFS(_utils.FSTree):
     """The file structure of the job queue data."""
 
-    def __init__(self, datadir):
+    def __init__(self, datadir: _utils.FSTree):
```

Comment:
This doesn't work, because [here](https://github.com/faster-cpython/tools/blob/main/PORTAL/jobs/__init__.py#L70) it is passed  `JobFS.requests`, [which is a `_utils.FSTree`](https://github.com/faster-cpython/tools/blob/main/PORTAL/jobs/_common.py#L243).

Reply:
Ah, right.